tests/k8s/client-w-conf.yml

```diff
@@ -31,12 +31,5 @@ spec:
             cpu: "1"
         ports:
         - containerPort: 9090
-        volumeMounts:
```

Comment:
if your doing changes unrelated to this pr, try to atleast split the commits. we want to be able to track down changes in the past.